pkg/server/validation.go

```diff
@@ -8,22 +8,25 @@ import (
 	"github.com/rancher/webhook/pkg/clients"
 	"github.com/rancher/webhook/pkg/resources/cluster"
 	"github.com/rancher/webhook/pkg/resources/clusterroletemplatebinding"
+	"github.com/rancher/webhook/pkg/resources/globalrole"
 	"github.com/rancher/webhook/pkg/resources/globalrolebinding"
 	"github.com/rancher/webhook/pkg/resources/projectroletemplatebinding"
 	"github.com/rancher/webhook/pkg/resources/roletemplate"
 	"github.com/rancher/wrangler/pkg/webhook"
 )
 
 func Validation(clients *clients.Clients) (http.Handler, error) {
+	clusters := cluster.NewValidator(clients.K8s.AuthorizationV1().SubjectAccessReviews())
```

Comment:
Why the reorder?

Reply:
I reordered so that the order they are declared matches the order the are mounted